components/gitpod-cli/cmd/rebuild.go

```diff
@@ -79,7 +79,7 @@ func runRebuild(ctx context.Context, supervisorClient *supervisor.SupervisorClie
 		fmt.Println("Alternatively, check out the following docs for getting started configuring your project")
 		fmt.Println("https://www.gitpod.io/docs/configure#configure-gitpod")
 		event.Set("ErrorCode", utils.RebuildErrorCode_MissingGitpodYaml)
-		return err
```

Comment:
I think we should return `false, nil` - indicating that there is no valid configuration but it is not system error.

if there is not valid configuration we should have top level catch all error code like user_error and exit with 1.

Reply:
now we return `<outcome>, <err>`